core/src/main/java/org/apache/calcite/sql/SqlCollation.java

```diff
@@ -356,8 +302,9 @@ public final Locale getLocale() {
 
   /**
    * @return Collator to be used for comparison, or null if no specific collator is defined
```

Comment:
The description here needs some modifications.

Reply:
Thanks for your feedback @XuQianJin-Stars , you are right.
I have re-worked the javadoc, let me know what you think.